docs/formats/csup.md

```diff
@@ -1,60 +1,58 @@
 ---
 sidebar_position: 4
-sidebar_label: VNG
+sidebar_label: Super Columnar
 ---
 
-# VNG Specification
+# Super Columnar Specification
 
-VNG, pronounced "ving", is a file format for columnar data based on
-[the Zed data model](zed.md).
-VNG is the "stacked" version of Zed, where the fields from a stream of
-Zed records are stacked into vectors that form columns.
+Super Columnar is a file format based on
+the [super data model](zed.md) where data is stacked to form columns.
```

Comment:
I simplified the opening lines a fair amount. A couple things I'll point out:

1. When I sat down to start working on this file, there were only three uses of the word "vector" left, and I think I recall hearing in the past that we wanted to phase that word out entirely if possible, so I've taken the liberty of trying to finish that off here.

2. Previously the text indicated that the columns were specifically for holding "a stream of records", but Super Columnar can handle streams of primitive values and other top-level non-record types, hence my deletion of that text.

But if there's a feeling I've gone too far here, please speak up!